.github/workflows/sphinx-build.yml

```diff
@@ -1,4 +1,4 @@
-name: Unittest Sphinx Build Deploy
+name: Flake8 Test Sphinx Build Deploy
```

Comment:
Flake, black, isort etc all fall into the testing bucket imo. They're verifications made against the code. Name is looking quite odd as is.